diff -Nur cctools-855-ld64-236.3/ld64/src/ld/Options.cpp cctools-855-ld64-236.3n/ld64/src/ld/Options.cpp
--- cctools-855-ld64-236.3/ld64/src/ld/Options.cpp	2014-06-30 02:35:58.378091873 +0800
+++ cctools-855-ld64-236.3n/ld64/src/ld/Options.cpp	2014-06-30 02:36:20.886091068 +0800
@@ -2380,6 +2380,10 @@
 			else if ( (strcmp(arg, "-ios_version_min") == 0) || (strcmp(arg, "-iphoneos_version_min") == 0) ) {
 				setIOSVersionMin(argv[++i]); 
 			}
+			else if ( (strcmp(arg, "-aspen_version_min") == 0) ) {
+                //for backward compitable with llvm-gcc.
+                ++i;
+			}
 			else if ( strcmp(arg, "-ios_simulator_version_min") == 0 ) {
 				setIOSVersionMin(argv[++i]);
 				fTargetIOSSimulator = true;
